agimus_controller_ros/controller_base.py

```diff
@@ -146,7 +147,11 @@ def solve(self):
             [sensor_msg.joint_state.position, sensor_msg.joint_state.velocity]
         )
         self.fill_buffer()
-        point = self.traj_buffer.get_points(1, self.point_attributes)[0]
+        if self.traj_buffer.get_size(self.point_attributes) > 0:
```

Comment:
what happen during the initialization?

Reply:
we waited to fill the buffer with 2 times the horizon size so no problem